File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.py
# Last updated: 07/04/2026, 11:38:54
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        ans = 0 
        c = 0 
        for i in nums :
            if i == 1 :
                c+=1
            else:
                ans = max(c,ans)
                c = 0
                

        return max(ans,c) 
        

File: 485-MaxConsecutiveOnes/485-MaxConsecutiveOnes.cpp
// Last updated: 07/04/2026, 11:38:50
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
       int count = 0 ;
       int ans = 0 ;
       for ( int i : nums){
        if ( i == 1){
            count ++;
        }
        ans = max(count,ans);
        if ( i == 0 ) count = 0;
       } return ans;
    }
};